pkg/didmethod/httpbinding/resolver.go

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package httpbinding
+
+import (
+	"crypto/tls"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"path"
+	"time"
+
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+	errors "golang.org/x/xerrors"
+)
+
+var logger = log.New("aries-framework/didmethod/httpbinding")
+
+// HTTPClientOpts holds options for the HTTP transport implementation of DIDResolver
+// it has a required http.Client instance and optional basePath which is appended to the remove DID Resolver URL
+type HTTPClientOpts struct {
```

Comment:
This struct should not be exported.

Reply:
Ah, I renamed it to make capital "HTTP" and did not notice the side effect. Thanks!